python/grass/jupyter/init-funcs.py

```diff
@@ -0,0 +1,23 @@
+import os
+import grass.script as gs
+import grass.script.setup as gsetup
+
+
+def init(gisbase, grassdata_filepath, location, mapset):
```

Comment:
Suggestions for a different name are welcome, but `init` is okay.

As of now, the GISBASE environmental variable needs to be defined before `import grass.script as gs`, so we don't need ask user to pass it here as we can just take it from the environment here and remove the `gisbase` parameter. This could be done in `grass.script.setup.init` too, but let's do it here now since we are sure what is our context here.

I'm not sure what to do with the db/loc/mapset trio, we had discussions in the past about passing just path to mapset versus the three variables. This might be the occasion to try that in a prominent part of the API.